dist/js/master.js

```diff
@@ -1408,12 +1414,7 @@ bcpl.navigation = function ($, keyCodes) {
 		}
 	};
 
-	$(document).on('mouseover', '.nav-and-search:not(.search-is-active) #responsive-sliding-navigation button, #responsive-sliding-navigation .submenu-wrapper', navigationMouseover);
-	$(document).on('mouseleave', '.nav-and-search:not(.search-is-active) #responsive-sliding-navigation button, #responsive-sliding-navigation .submenu-wrapper', navigationMouseleave);
-	$(document).on('keydown', '#responsive-sliding-navigation button', navigationButtonKeyPressed);
-	$(document).on('keydown', '#responsive-sliding-navigation', navigationKeyPressed);
-	$(document).on('click', navButtonSelector, navButtonClicked);
-	$(document).on('keydown', '#responsive-sliding-navigation a', navigationMenuItemKeyPressed);
+	$(document).on('mouseover', '.nav-and-search:not(.search-is-active) #responsive-sliding-navigation button, #responsive-sliding-navigation .submenu-wrapper', navigationMouseover).on('mouseleave', '.nav-and-search:not(.search-is-active) #responsive-sliding-navigation button, #responsive-sliding-navigation .submenu-wrapper', navigationMouseleave).on('keydown', '#responsive-sliding-navigation button', navigationButtonKeyPressed).on('keydown', '#responsive-sliding-navigation', navigationKeyPressed).on('click', navButtonSelector, navButtonClicked).on('click', backButtonSelector, onBackButtonClicked).on('keydown', '#responsive-sliding-navigation a', navigationMenuItemKeyPressed);
```

Comment:
I feel like I've already reviewed this change.

But in case I haven't, add some whitespace for God's sake.

Reply:
hey @ascire take a gander at the file location on your comment 😆

Reply:
...Welp.